src/app/home/DetailViewComponent.tsx

```diff
@@ -152,24 +163,29 @@ class DetailViewComponent extends Component<IProps, IState> {
           <ClusterContext.Provider value={{ watchClusterAddEditStatus }}>
             <ClusterDataListItem
               dataList={allClusters}
-              id="clusterList"
+              id={DataListItems.clusterList}
```

Comment:
I don't think we want our html ids being something like "0". This should remain a descriptive unique string.